src/components/ReportActionItem/IOUQuote.js

```diff
@@ -45,7 +45,7 @@ const IOUQuote = props => (
                 focusable={props.shouldAllowViewDetails}
             >
                 <Text>
-                    <Text style={[styles.chatItemMessageLink]}>
+                    <Text style={props.shouldAllowViewDetails ? [styles.chatItemMessageLink] : []}>
```

Comment:
**NAB:** We don't have to pass array. We can use 
```suggestion
                    <Text style={props.shouldAllowViewDetails && styles.chatItemMessageLink}>
```

Reply:
Done!